kotlin-css/src/main/kotlin/kotlinx/css/TagSelectors.kt

```diff
@@ -2,6 +2,8 @@ package kotlinx.css
 
 class TagSelector(val tagName: String)
 
+val star get() = TagSelector("*")
```

Comment:
It's not actually a tag

Reply:
I think you should rather add following member function to https://github.com/JetBrains/kotlin-wrappers/blob/master/kotlin-css/src/main/kotlin/kotlinx/css/CSSBuilder.kt#L34

```
fun star(block: RuleSet) = "*"(block)
```

The "star" name is also disputable. I would prefer something more semantic, like "everything" or "wildcard"

Reply:
Jeez, what was I thinking?